### PR TITLE
Code Quality: Merged all interop into Win32PInvoke

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

Just moved. Nothing has been modified other than formatting or adding extra references to avoid ambiguity.

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   _None_

## Screenshots

_None_

### CODE REVIEW

**Comment by 0x5bfa** [comment]
This is the last time touching around interop classes.
Deleted all interop classes and merged into single class, so we not can gradually replace them with CsWin32 without conflict (there were some classes that is the same definition) in the single place.

It's a bit large but ready for review.

File: src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs [hunk 1]
```diff
@@ -154,14 +154,14 @@ public static IAsyncOperation<BaseStorageFile> FromPathAsync(string path)
...
-			using var hFile = NativeFileOperationsHelper.OpenFileForRead(path);
+			using var hFile = Win32Helper.OpenFileForRead(path);
 			return !hFile.IsInvalid;
 		}
 
 		private static bool IsNativePath(string path)
 		{
 			var isShortcut = FileExtensionHelpers.IsShortcutOrUrlFile(path);
-			var isAlternateStream = RegexHelpers.AlternateStream().IsMatch(path);
+			var isAlternateStream = System.Text.RegularExpressions.Regex.IsMatch(path, @"\w:\w");
```
Comment (gumbarros) [inline src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs:164]: Why the `RegexHelpers` class was removed?

**Review by gumbarros — CHANGES_REQUESTED** [review]
LGTM except for this

File: src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs:164]: Omg, must be pull problem. I'll revert them.

Comment (gumbarros) [reply src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs:164]: I checked all the files, and I'm nearly sure it's just here that is messed

**Review by gumbarros — APPROVED** [review]
LGTM

**Pull request merged**